recipe/meta.yaml

```
@@ -69,3 +69,6 @@ about:
 extra:
   recipe-maintainers:
     - sugatoray
```

Consider adding the following comment to make it clear which maintainers are from upstream (`docarray` repository).

```suggestion
    - sugatoray
    # Maintainers from package (upstream) repository
```